cmake/modules/RootBuildOptions.cmake

```
@@ -132,6 +132,7 @@ ROOT_BUILD_OPTION(fortran OFF "Build Fortran components of ROOT")
 ROOT_BUILD_OPTION(gdml ON "Enable support for GDML (Geometry Description Markup Language)")
 ROOT_BUILD_OPTION(gnuinstall OFF "Perform installation following the GNU guidelines")
 ROOT_BUILD_OPTION(gviz OFF "Enable support for Graphviz (graph visualization software)")
+ROOT_BUILD_OPTION(html OFF "Enable support for HTML Gui")
```

```suggestion
ROOT_BUILD_OPTION(html OFF "Enable support for THtml, the legacy ROOT documentation system (deprecated)")
```
I would be more specific here, also about the deprecation.

See also: https://root.cern.ch/doc/master/classTHtml.html